packages/mobile/src/web3/reducer.ts

```diff
@@ -5,9 +5,6 @@ import { Actions, ActionTypes, Web3SyncProgress } from 'src/web3/actions'
 export interface State {
   syncProgress: Web3SyncProgress
   latestBlockNumber: number
-  account: string | null
```

Comment:
Why move these?

Reply:
good question
I guess it wasn't strictly necessary but wanted to make clear that web3 was no longer managing the account state

Reply:
I appreciate that. But if you're going to move it, you'll also need to create a redux migration for existing users on alfajores, and a test for that migration. It's not a lot of work but it'll further increase conflicts with my PR here:
https://github.com/celo-org/celo-monorepo/pull/3777
Which will merge hopefully in a day or two

Reply:
reverted this change